airgap/v2/cmd/client/download/download.go

```diff
@@ -191,7 +183,8 @@ func (dc *DownloadConfig) downloadFile(fn string, fid string) error {
 		return fmt.Errorf("error while writing data to the output file: %v", err)
 	}
 
-	log.Info("File downloaded successfully!", "name/id", name)
+	dc.log.Info("File downloaded successfully!", "name/id", name)
+	dc.log.Info("Checksum", "checksum", cs)
```

Comment:
Let's make this a single line log
```suggestion
	dc.log.Info("File downloaded successfully!", "name/id", name, "checksum", cs)
```